src/main/java/org/junit/internal/runners/TestMethod.java

```diff
@@ -48,7 +49,7 @@ public long getTimeout() {
         }
     }
 
-    boolean isUnexpected(Throwable exception) {
+    boolean isUnexpected(@Nullable Throwable exception) {
```

Comment:
Why @Nullable here? Won't it cause potential NPE when exception.getClass() is called later?

Reply:
Thanks for pointing it out! We have a new branch also comment about why the annotations are added, once its finished, we will correct this according to that one after its peer review!